rcsb/workflow/targets/ProteinTargetSequenceWorkflow.py

```diff
@@ -474,7 +479,7 @@ def __buildCofactorData(self, referenceResourceName, resourceName, backup=False,
                 aP = PharosTargetCofactorProvider(cachePath=self.__cachePath, useCache=True, useStash=True, useGit=True)
                 ok = aP.buildCofactorList(resultPath, crmpObj=crmpObj, lnmpObj=lnmpObj, maxActivity=maxActivity)
                 if backup:
-                    okB = aP.backup(self.__cfgOb, self.__configName, remotePrefix=remotePrefix)
+                    okB = aP.backup(self.__cfgOb, self.__configName, remotePrefix=remotePrefix, useGit=True)
```

Comment:
This was the only file that wasn't updating in GitHub without calling DictMethodResourceProvider.syncCache. We may decide to stop updating to git, but that can be a separate decision.

Reply:
> Just to clarify, you mean this file ([Pharos-cofactors](https://github.com/rcsb/py-rcsb_exdb_assets_stash/tree/development/stash/Pharos-cofactors))? And this change will cause it to be updated every week on GitHub (whenever the new Luigi cache-rebuilding task runs)?
Yes.

> Will all other files be updated on [py-rcsb_exdb_assets_stash](https://github.com/rcsb/py-rcsb_exdb_assets_stash/tree/development/stash) every week too?
Yes. As it stands now, anyway, that was the behavior for the update scripts. `DictMethodResourceProvider.syncCache()` was actually creating duplicate commits for the others that it was syncing, so it makes sense to not call that and just take care of it here.

Reply:
OK cool, thanks for confirming and explaining.